### DISCUSSION TITLE
RL/Sequential Decision Making Birds of Feather session at JuliaCon 2021

### DISCUSSION THREAD

**Discussion by zsunberg** [JuliaReinforcementLearning/ReinforcementLearning.jl#220] [Ideas]
Hi Everyone,

Would there be any interest in organizing a Birds of Feather session for reinforcement learning in Julia at JuliaCon 2021? (https://pretalx.com/juliacon2021/cfp) Or has anyone done this? I'd be happy to submit a proposal if no-one else has. Would be great to sit down and have a conversation with everyone :)

@jonathan-laurent, @findmyway 

- Zach

**Comment by jonathan-laurent** [comment]
I was just about to write a post to test the interest for a BoF session focused on RL. What a coincidence!

I would be excited to participate to such a session. :-)

**Reply by findmyway** [reply]
Haha, I'm also interested in such a session!

I guess @jonathan-laurent  already has some draft ideas. How about adding them somewhere for public access so that we can collect them all and submit one?

**Comment by Sid-Bhatia-0** [comment]
This sounds great! I'd be happy to attend such a session :)

**Comment by zsunberg** [comment]
Great, I will look at the proposal form more closely tomorrow. I am not sure what's needed. If anyone has any suggestions, let me know.

**Reply by zsunberg** [reply]
Here is the abstract I proposed:

> Reinforcement learning (RL) is one of the most important rising sub-fields of artificial intelligence. Julia's flexibility and speed make it uniquely well-suited to RL. There are several packages designed for RL or planning in Julia including ReinforcementLearning.jl, POMDPs.jl, and AlphaZero.jl, and Julia is well-suited to RL even without specialized packages. Package maintainers and anyone interested will discuss best practices, accomplishments, and how we can make RL in Julia more accessible.

I can edit it until 11:59 UTC

**Reply by Sid-Bhatia-0** [reply]
Looks good to me. Thank you for taking the initiative! 😄

**Reply by jonathan-laurent** [reply]
This is a nice abstract. Thanks Zachary for submitting this proposal!

**Reply by zsunberg** [reply]
I invited @jonathan-laurent and @findmyway to be "speakers". @Sid-Bhatia-0 if you send me an email, I can invite you as well.

**Reply by zsunberg** [reply]
https://pretalx.com/juliacon2021/talk/review/3DALH7FUU88FCJ9EB3BMTRX9U988UYMK if anyone wants to see it
